src/utils/settings.py

```diff
@@ -13,23 +14,49 @@ def is_running_pytest() -> bool:
     return "pytest" in sys.modules
 
 
-project_root = os.path.dirname(os.path.abspath(__file__))
+def load_env_from_json() -> None:
+    """Load the configuration from the config.json file."""
+    config_path = os.getenv(
+        "AICORE_HOME",
+        os.path.abspath(
+            "../../go/src/github.com/kyma-project/kyma-companion/config/config.json"
+        ),
+    )  # Default to root config.json.
+
+    if os.path.exists(config_path):
+        try:
+            # Load the configuration from the given path and set the environment variables.
+            with open(config_path) as file:
+                config_file = json.load(file)
+
+                # Set environment variables for all keys except "models"
+                for key, value in config_file.items():
+                    if key != "models":  # Skip models
+                        os.environ[key] = str(value)
```

Comment:
please add the following error handling too:
```
     except json.JSONDecodeError as e:
       logging.error(f"Invalid JSON format in config file {config_path}: {e}")
       raise
```

Reply:
done